jest.config.transform.svg.js

```diff
@@ -1,13 +1,22 @@
 /* eslint-disable import/no-commonjs, no-unused-vars, prefer-template */
 
+const fs = require('fs');
 const path = require('path');
 
 module.exports = {
   process(src, filename, config, options) {
     // import iconCautionSvg from 'src/svgs/icon-caution-fill.svg';
```

Comment:
Given that this is commented out, I think we can safely remove these comments. If we need that code in someway, it's always in our version control system.

Unless `process` processes the code included in the block, which if that's the case, don't worry about this comment.

Reply:
This is a little bit of tricky configuration -- it's an area of our codebase that is poorly understood. I am opting to leave the code in for future Jest config tweakers -- it has been a